frontend/exporter/adt-into/src/lib.rs

```diff
@@ -226,6 +226,11 @@ fn variant_to_arm(
         // TODO: refactor
```

Comment:
While you're here ... Can you add more information here and an issue or remove it?

Reply:
I removed those, I don't recall what I meant by that :/